### DISCUSSION TITLE
Consulta Test de Integración - Actividad 3

### DISCUSSION THREAD

**Discussion by renatacatelli** [frlp-utn-ingsoft/catedra#124] [Q&A] [answered]
Buenas,

Estaba haciendo algunos de los tests de integración y me salta un error con la KEY de la bd. Si mal no recuerdo también me pasó algo similar en el de unidad. No termino de entender por qué se dan estos errores, como se relaciona con el test? 

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/0848f2f0-28fe-42e3-8b30-000b49900020)

Gracias!!

**Comment by RodrigoJacznik** [comment, marked as answer]
Buenas, el error es similar al anterior. Fíjate que en tu .env tengas algún valor en SECRET_KEY.

**Reply by renatacatelli** [reply]
Buenas! Disculpa la demora en responder. Si gracias, ya le di un valor y ahora no me da más ese error. Mil gracias
